File: Eclipse/VersaoComBanco/ProjetoFinalDm102/src/br/inatel/mobile/pos/dm102/trabalhofinal/model/ClienteFisico.java
package br.inatel.mobile.pos.dm102.trabalhofinal.model;

public class ClienteFisico extends Cliente {

	private String cpf;
	private String identidade;
	private String tipoDaIdentidade;
	
	public ClienteFisico(String nome, String endereco, String telefone, String cpf, String identidade,
			String tipoDaIdentidade) {
		super(nome, endereco, telefone);
		this.cpf = cpf;
		this.identidade = identidade;
		this.tipoDaIdentidade = tipoDaIdentidade;
	}

	public String getCpf() {
		return cpf;
	}
	
	public String getIdentidade() {
		return identidade;
	}
	
	public String getTipoDaIdentidade() {
		return tipoDaIdentidade;
	}

	@Override
	public String toString() {
		return super.toString() + " -- CPF: " + cpf + " -- Identidade: " + identidade + " -- Tipo da Identidade: " + tipoDaIdentidade;
	}

}

File: DM102/VisualStudio/ProjetoFinalDm102/ProjetoFinalDm102/ClienteFisico.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjetoFinalDm102
{
    class ClienteFisico : Cliente
    {
        private string cpf;
        private string identidade;
        private string tipoDeIdentidade;

        public ClienteFisico(string nome, string endereco, string telefone, string cpf, string identidade, string tipoDeIdentidade) : base(nome, endereco, telefone)
        {
            this.cpf = cpf;
            this.identidade = identidade;
            this.tipoDeIdentidade = tipoDeIdentidade;
        }

        public string GetCpf()
        {
            return cpf;
        }

        public string GetIdentidade()
        {
            return identidade;
        }

        public string GetTipoDeIdentidade()
        {
            return tipoDeIdentidade;
        }

        public override string ToString()
        {
            return base.ToString() + " - CPF: "+ cpf + " - Identidade: " + identidade + " - Tipo de Identidade: " + tipoDeIdentidade;
        }

    }
}
